<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="MainOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>781</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSouth">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 0);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Go South</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushWest">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 0);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Go West</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushEast">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>80</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 0);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Go East</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushNorth">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 0);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Go North</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushMap">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 0);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Display Map</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushTake">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 0);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Pickup Item</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushInteract">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 0);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Interact</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushTeleport">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 0);
background-color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>Teleport</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushQuit">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 0);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Quit Game</string>
    </property>
   </widget>
   <widget class="QLabel" name="ImageOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>361</width>
      <height>211</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ExceptionLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="WordleInput">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>250</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="WordleLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>391</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="finallabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>801</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Game Over</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>MainOutput</zorder>
   <zorder>pushSouth</zorder>
   <zorder>pushWest</zorder>
   <zorder>pushEast</zorder>
   <zorder>pushNorth</zorder>
   <zorder>pushTake</zorder>
   <zorder>pushInteract</zorder>
   <zorder>pushTeleport</zorder>
   <zorder>pushQuit</zorder>
   <zorder>ImageOutput</zorder>
   <zorder>pushMap</zorder>
   <zorder>ExceptionLabel</zorder>
   <zorder>WordleInput</zorder>
   <zorder>WordleLabel</zorder>
   <zorder>finallabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
